permissions/component-kohsuke-maven-plugins.yml

```diff
@@ -4,5 +4,6 @@ name: "dummy-unused-artifact-name"
 paths:
   - "org/jvnet"
   - "org/kohsuke"
+  - "io/jenkins/tools/maven"
```

Comment:
might need to move this to `permissions/component-license-maven-plugin.yml` ? as won't it need to be cd enabled?

Reply:
Doesn't "- "io/jenkins/tools/maven/license-maven-plugin"" suffice in the license-maven-plugin.yml?

Reply:
no maven plugins also update the groupid IIRC